src/components/AddCoin/AddCoin.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { View, Text, SafeAreaView, TextInput, Pressable, TouchableOpacity } from 'react-native';
+import { addCoinStyles } from './addCoinStyles';
+
+const AddCoin = ({ navigation }) => {
+    const [coinTicker, setCoinTicker] = useState<string>('');
```

Comment:
type declaration unnecessary:

useState('')

if you declare a initial value, the state will have the type of the default value already